test/SlowTests/Server/Documents/ETL/Raven/RavenDB_9072.cs

```diff
@@ -86,7 +86,7 @@ public async Task CanTestScript()
 
                         Assert.Equal(0, result.TransformationErrors.Count);
 
-                        Assert.Equal(5, result.Commands.Count);
+                        Assert.True(result.Commands.Count >= 4);
```

Comment:
Why it changed?

Reply:
Because we don't send the delete doc operation automatically when there is a script.

Reply:
Then why not `Assert.Equal(4, result.Commands.Count);` ?

Reply:
I don't like tests that test the implementation and demand update on implementation changes.
There are three loads in the script so actually as I see it it will be better to just check for 3 put commands.
But if you prefer I will use `Equal` here.

Reply:
I prefer to see a failing assertion and fix it once the implementation changes than silently pass an assertion while the original condition is no longer true.

Given the example here `result.Commands.Count >= 4` it's very risky because if someone make a mistake and will start sending each ETL command twice then `result.Commands.Count: 8` won't make the test to fail.